### ISSUE TITLE
Error installing ethik with pip

### ISSUE THREAD

**Issue by Vayel** [XAI-ANITI/ethik#124]
```
$ pip install ethik
Collecting ethik
  Using cached https://files.pythonhosted.org/packages/73/34/f866ebb5e4076ac5be04d5707c13cb2039cf13f64fc9b83d9f6bd285a642/ethik-0.0.3.tar.gz
    ERROR: Command errored out with exit status 1:
     command: /home/vincent/.local/share/virtualenvs/ethik-ui/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-2rfmtr4d/ethik/setup.py'"'"'; __file__='"'"'/tmp/pip-install-2rfmtr4d/ethik/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-install-2rfmtr4d/ethik/pip-egg-info
         cwd: /tmp/pip-install-2rfmtr4d/ethik/
    Complete output (5 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-2rfmtr4d/ethik/setup.py", line 32, in <module>
        with io.open(os.path.join(here, "requirements.txt"), encoding="utf-8") as f:
    FileNotFoundError: [Errno 2] No such file or directory: '/tmp/pip-install-2rfmtr4d/ethik/requirements.txt'
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

@MaxHalford

**Comment by Vayel** [comment]
I tried something in #123

**Comment by Vayel** [comment]
@MaxHalford should we fix 0.0.3, or having 0.0.4 is sufficient? I'm not confortable with having a non-working release on Pypi.

**Comment by MaxHalford** [comment]
We can release version `0.0.4` and remove version `0.0.3` from Pypi.

**Comment by Vayel** [comment]
v0.0.4 being released, can you remove v0.0.3 from Pypi?
